### PR TITLE
Debt feature branch

### PR DESCRIPTION
**Pull request by yenanw**
Co-authored-by: gabrielbrattgard <gabriel.brattgard@gmail.com>

### CODE REVIEW

**Review by OscarSanner — CHANGES_REQUESTED** [review]
I think this looks really nice good. Time effective too. Just one general question to all.

@yenanw @gabrielbrattgard @Puh00 @OlofSjogren 

Will we do this type of sub-package division?

File: app/src/main/java/com/goayo/debtify/model/debt/Ledger.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+
+/**
+ * @author Gabriel Brattgård, Yenan Wang
+ * @date 2020-09-15
+ * <p>
+ * Ledger holds a list DebtTracker and manages debts and payments
+ */
+public class Ledger {
+}
```
Comment (OscarSanner) [inline app/src/main/java/com/goayo/debtify/model/debt/Ledger.java:11]: Both team @gabrielbrattgard @yenanw  and team @OlofSjogren @Puh00 Created a ledger class.

**Review by OscarSanner — CHANGES_REQUESTED** [review]

**Comment by yenanw** [comment]
I think sub-package division is quite nice. Can always refactor it later if found not suitable. @OscarSanner

**Review by OscarSanner — APPROVED** [review]
Looking good!

**Pull request merged**